package.json

```diff
@@ -23,6 +23,7 @@
     "chai-datetime": "^1.8.0",
     "cookie-parser": "^1.4.5",
     "cors": "^2.8.5",
+    "curl": "^0.1.4",
```

Comment:
Please remove this and, in order to update the `package-lock.json` file as well, re-run `npm install`. The `dependencies` section is only for libraries used in the code. My guess is that you were using this on the command line for testing, right?

Reply:
Yup, I'll remove this